lib/iris/tests/unit/util/test_combine_cubes.py

```diff
@@ -0,0 +1,139 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Unit tests for the :func:`iris.io.loading.combine_cubes` function.
+
+Note: These tests are fairly extensive to cover functional uses within the loading
+operations.
+TODO: when function is public API, extend testing to the extended API options,
```

Comment:
I believe this TODO is completed and can be removed now.

Reply:
OK thanks!
[811168b](https://github.com/SciTools/iris/pull/6334/commits/811168bf9674023e330edf1f2e153b282a91cae0)